docs/modules/ROOT/pages/how-tos/testing/integration/debugging.adoc

```diff
@@ -0,0 +1,44 @@
+= Debugging Integration Tests
+
+In the event of an integration test scenario failing, there are a number of steps to take in order to debug the issue before rerunning the test in question. These steps are outlined below.
+
+
+.Prerequisites
+- You have a snapshot that has completed all its initial tests. All initial tests must be finished before attempting to debug the failed test.
+
+- You have access to the console in order to view the logs of the failed test.
+
+.Procedure
+
+. Identify the IntegrationTestScenario that has failed. This can be found in the console by navigating to 
+Applications/your_application_name/activity and select "Pipeline Runs" under "Activity By". Your view should be similar to the below image.
+
++
+image::activityview.png[role="border" alt="Activity View"]
+
+. Click on the pipeline run that has a status of "Failed". This will take you to a detailed view of the failed run. On the right side of this page, look for the "Log Snippet" section. This will contain the most recent error returned by the integration test. For more comprehensive logs, click on "See Logs". See the below image for reference.
+
++
+image::testfailed.png[role="border" alt="Test Failed"]
+
+. In the above example the output suggests that the pipelineRun is missing an expected parameter, "username". To edit the integration test, scroll down to the "Integration Test" field and click on the listed test. This should take you to a view similar to the one seen below.
+
++
+image::testview.png[role="border" alt="Test View"]
+
+. Click on the "Actions" button seen above and select "Edit". This should take you to the below view.
+
++
+image::edittest.png[role="border" alt="Edit Test"]
+
+. In this example we will need to click on the "Parameters" dropdown and add the parameter expected by the pipeline run.
+In this case it's "username" and we will set the value to "testuser". Once you've completed your edits, click "Save Changes" at the bottom of the screen to update the integration test. 
+
+. Return to the activity view mentioned in step 1 and click on the 3 verticle dots on the right side of the pipeline run. Within this expanded menu, click "rerun". This will rerun the integration test with our updated parameter. This should result in a success as seen in the below image. For a more comprehensive, CLI-focused explanation of rerunning tests, click xref:/how-tos/testing/integration/rerunning.adoc[here].
```

Comment:
Can we instead add information on how to rerun from the UI to https://konflux-ci.dev/docs/how-tos/testing/integration/rerunning/? Then we can just simply link users there to rerun the test.

Reply:
Yeah that makes total sense. I can add the supporting text to those steps as an alternate approach to CLI. I was wondering though, is there an argument in situations like this for a separate subpage for CLI and UI respectively? To me that would provide a bit more clarity for a user.